README.md

```diff
@@ -109,29 +109,27 @@ efficient format. Input files should be sorted by ascending time.
 
 |Name|Unit|Type|Description|
 |---|---|---|---|
-| obs_id | None | str |Unique observation ID for the observation |
+| obs_id | None | str | Unique observation ID for the observation |
 | exposure_id | None | str | Exposure or Image ID from which observation was measured |
-| mjd | days | float | MJD of the observation in UTC** |
+| mjd | days | float | MJD of the observation in UTC[^1] |
 | ra  | degree | float | Right Ascension (J2000) |
 | dec  | degree | float | Declination (J2000) |
-| ra_sigma  | degree | float | 1-sigma uncertainty in Right Ascension |
-| dec_sigma  | degree | float | 1-sigma uncertainty in Declination |
+| ra_sigma  | degree | float | 1-sigma uncertainty in Right Ascension (Nullable)[^2] |
+| dec_sigma  | degree | float | 1-sigma uncertainty in Declination (Nullable)[^2] |
```

Comment:
I don't think "Nullable" is quite an accurate description. They're still always floats, they aren't ever valued as `None`. It's more than NaN is explicitly supposed to be expected.

Reply:
How about swapping to "Optional"? I can update the footnote to say that the column should still be included in the CSV but it should be empty. 

(We could relax this too and make the presence of the column actually optional.)

Reply:
Yeah, I think "optional" is a better word for it.